vagrant/vagrant-test.sh

```diff
@@ -21,6 +21,10 @@ fi
 
 pushd /build || { echo >&2 "Can't pushd to /build"; exit 1; }
 
+# Trick the QEMU-related test functions, so it appears we're not already
+# running under KVM
+sed -i 's/systemd-detect-virt -v/echo not-kvm/g' test/test-functions
```

Comment:
I imagine this goes away once systemd/systemd#13666 is merged, right?

Reply:
Yes, that's only for debugging.